pkg/handler/aws/participationAggregation/aggregation.go

```diff
@@ -4,32 +4,30 @@ import (
 	"context"
```

Comment:
I don't understand how this file may be used in this pull request. I thought the controller was mostly part of the old architecture, but I am not sure.

Reply:
This is used by the handler in `./main.go`.  
I can merge those two files if you want, but I thought it made the main file clearer to identify as the lambda handler.  
As you can see `func NewParticipationAggregationHandler` now uses the repository and service.

Reply:
Nah it's completely fine. I just didn't understand in the beginning, but now that you are pointing it out it makes sense